src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -304,11 +305,13 @@ public TextClassificationTransformer Fit(IDataView input)
                     if (_options.ValidationSet != null)
                         trainer.Validate(pch, ch, i);
                 }
-                _transformer = new TextClassificationTransformer(_host, _options, trainer.Model, trainer.Tokenizer.Vocabulary);
+                var labelCol = input.Schema.GetColumnOrNull(_options.LabelColumnName);
 
-                _transformer.GetOutputSchema(input.Schema);
+                transformer = new TextClassificationTransformer(_host, _options, trainer.Model, new DataViewSchema.DetachedColumn(labelCol.Value));
```

Comment:
>labelCol [](http://example.com/codeflow?start=130&length=8)

is it possible this can be null? The call GetColumnOrNull suggests it can return null.

Reply:
Yes, and I thought I was checking before that call but it doesn't look like I am. Good catch, let me add that.

Reply:
Ok, I have added a check in for all the input schema, so that line will never return null since an error will be thrown prior if there is a problem.